music/src/core/chords_test.ts

```diff
@@ -46,6 +49,8 @@ test('Test Chord Root', (t: test.Test) => {
   t.equal(ChordSymbols.root('Abmaj7'), 8);
   t.equal(ChordSymbols.root('D##5(add6)'), 4);
   t.equal(ChordSymbols.root('F(b7)(#9)(b13)'), 5);
+  t.throws(() => ChordSymbols.root(''), ChordSymbolException);
+  t.throws(() => ChordSymbols.root('o'), ChordSymbolException);
```

Comment:
I would also add a test for our friend "M" here

Reply:
Done.